source/material_model/rheology/frank_kamenetskii.cc

```diff
@@ -71,6 +85,15 @@ namespace aspect
                            "for a total of N+1 values, where N is the number of all compositional fields or only "
                            "those corresponding to chemical compositions. "
                            "If only one value is given, then all use the same value.  Units: None");
+
+        // new addition
+        prm.declare_entry ("Pressure prefactors for Frank Kamenetskii", "4.159",
```

Comment:
Can you make the default value `0` so that the output of all current tests should not change?